supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -839,6 +811,16 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 *See also*:
 
+[[rolling-stream]]
+==== image:images/yes.png[yes] Rolling Stream (noun)
+*Description*: _Rolling Streams_, also referred to as Rolling Application Streams, include Red Hat Enterprise Linux tools and applications that are updated frequently. Later versions of these streams replace earlier versions. Therefore, only one version at a time is supported.
```

Comment:
ditto my comment above about defining a singular term in plural form. again, if we choose singular, change:

A _Rolling Stream_, also referred to as a Rolling Application Stream, includes..../?